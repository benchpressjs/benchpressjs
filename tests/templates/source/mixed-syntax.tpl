<div id="container">
<!-- IF rooms.length -->
  <!-- BEGIN rooms -->
    <!-- IF !rooms.private -->
      <a data-func="webrtc.joinRoom" data-room="{rooms.slug}" href="#" class="list-group-item">
        <h4 class="list-group-item-heading">{rooms.name}</h4>
        <p class="list-group-item-text">{rooms.description}</p>
      </a>
    <!-- ENDIF !rooms.private -->
  <!-- END rooms -->
<!-- ELSE -->
  <a data-func="webrtc.newRoom" href="#" class="list-group-item">
    <h4 class="list-group-item-heading">No rooms currently available!</h4>
    <p class="list-group-item-text">Click here to create one!</p>
  </a>
<!-- ENDIF rooms.length -->
</div>


{{{if rooms.length}}}
  <!-- BEGIN rooms -->
    {{{if !rooms.private}}}
      <a data-func="webrtc.joinRoom" data-room="{rooms.slug}" href="#" class="list-group-item">
        <h4 class="list-group-item-heading">{rooms.name}</h4>
        <p class="list-group-item-text">{rooms.description}</p>
      </a>
    <!-- ENDIF !rooms.private -->
  {{{ end }}}
<!-- ELSE -->
  <a data-func="webrtc.newRoom" href="#" class="list-group-item">
    <h4 class="list-group-item-heading">No rooms currently available!</h4>
    <p class="list-group-item-text">Click here to create one!</p>
  </a>
<!-- ENDIF rooms.length -->