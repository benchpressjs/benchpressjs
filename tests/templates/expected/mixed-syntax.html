<div id="container">
      <a data-func="webrtc.joinRoom" data-room="test" href="#" class="list-group-item">
        <h4 class="list-group-item-heading">Test</h4>
        <p class="list-group-item-text">testing</p>
      </a>
</div>


      <a data-func="webrtc.joinRoom" data-room="test" href="#" class="list-group-item">
        <h4 class="list-group-item-heading">Test</h4>
        <p class="list-group-item-text">testing</p>
      </a>